tests/testthat/_snaps/hypothesize.md

```diff
@@ -33,6 +33,15 @@
       Warning:
       Parameter values are not specified when testing that two variables are independent.
 
+# hypothesize() throws a warning when params are set with paired independence
+
+    Code
+      res_ <- mtcars_df %>% specify(response = mpg) %>% hypothesize(null = "paired independence",
+        mu = 25)
+    Condition
+      Warning:
+      Parameter values are not specified when testing paired independence.
```

Comment:
Should this say "are not" or "should not be" specified?

Reply:
I like your suggestion better! Added that in [c3ec354](https://github.com/tidymodels/infer/pull/494/commits/c3ec354acb63af38c2795f70fcff48097339a7eb), and also changed the existing warning that that pattern was copied from.